addons/web/static/src/views/calendar/calendar_arch_parser.js

```diff
@@ -0,0 +1,198 @@
+
+                    const field = fields[fieldName];
+                    if (!node.hasAttribute("invisible") || node.hasAttribute("filters")) {
+                        let filterInfo = null;
+                        if (
+                            node.hasAttribute("avatar_field") ||
+                            node.hasAttribute("write_model") ||
+                            node.hasAttribute("write_field") ||
+                            node.hasAttribute("color") ||
+                            node.hasAttribute("filters")
+                        ) {
+                            filtersInfo[fieldName] = filtersInfo[fieldName] || {
+                                avatarFieldName: null,
+                                colorFieldName: null,
+                                fieldName,
+                                filterFieldName: null,
+                                label: field.string,
+                                resModel: field.relation,
+                                writeFieldName: null,
+                                writeResModel: null,
+                            };
+                            filterInfo = filtersInfo[fieldName];
+                        }
+                        if (node.hasAttribute("filter_field")) {
+                            filterInfo.filterFieldName = node.getAttribute("filter_field");
+                        }
+                        if (node.hasAttribute("avatar_field")) {
+                            filterInfo.avatarFieldName = node.getAttribute("avatar_field");
+                        }
+                        if (node.hasAttribute("write_model")) {
+                            filterInfo.writeResModel = node.getAttribute("write_model");
+                        }
+                        if (node.hasAttribute("write_field")) {
+                            filterInfo.writeFieldName = node.getAttribute("write_field");
+                        }
+                        if (node.hasAttribute("filters")) {
+                            if (node.hasAttribute("color")) {
+                                filterInfo.colorFieldName = node.getAttribute("color");
+                            }
+                            if (node.hasAttribute("avatar_field") && field.relation) {
+                                if (
+                                    field.relation.includes([
+                                        "res.users",
+                                        "res.partners",
+                                        "hr.employee",
+                                    ])
+                                ) {
+                                    filterInfo.avatarFieldName = "image_128";
+                                }
+                            }
+                        }
+                    }
+
+                    break;
+                }
+            }
+        });
+
+        return {
+            canCreate,
+            canDelete,
+            canEdit,
```

Comment:
in other view, I think we regroup those in `activeActions`

Reply:
The `activeActions` creates edit, create, delete and duplicate searching on the attributes with the same names.
Here we only have two of these (create and delete), the edit is completely different
```js
canEdit = !fields[fieldMapping.date_start].readonly;
```
Do you think that we need to use `activeActions`?